mm2src/coins/nft/storage/sql_storage.rs

```diff
@@ -499,19 +483,19 @@ fn block_number_from_row(row: &Row<'_>) -> Result<i64, SqlError> { row.get::<_,
 
 fn nft_amount_from_row(row: &Row<'_>) -> Result<String, SqlError> { row.get(0) }
 
-fn get_nfts_by_token_address_statement(conn: &Connection, table_name: String) -> MmResult<Statement, SqlError> {
+fn get_nfts_by_token_address_statement(conn: &Connection, table_name: String) -> Result<Statement, SqlError> {
     let sql_query = format!("SELECT * FROM {} WHERE token_address = ?", table_name);
```

Comment:
A note from ChatGTP on using `SELECT *` 🙂 

Using `SELECT *` in SQL can be safe in some cases but may also have potential drawbacks. Here are some considerations:

Pros of using `SELECT *`:
1. Simplicity: It's a convenient way to retrieve all columns from a table without specifying each column explicitly.
2. Future-proofing: If the table structure changes (e.g., columns are added or removed), `SELECT *` will automatically adapt to the changes, so your queries won't break.

Cons of using `SELECT *`:
1. Performance: Selecting all columns, especially from large tables, can lead to unnecessary data transfer and increased query execution time. This can have a negative impact on performance, especially if you're only interested in a subset of the columns.
2. Network bandwidth: If you're transferring data over a network (e.g., from a database server to a client application), selecting all columns can consume more bandwidth than necessary, potentially slowing down data retrieval.
3. Maintenance: It can make your code less maintainable, as it may be unclear which columns are being used in a query, making it harder to understand the query's purpose.
4. Security: If sensitive or unnecessary data is included in the result set, it could expose more information than intended, potentially leading to security risks, especially if access control is not properly configured.

To mitigate these drawbacks, consider the following best practices:

1. Explicitly specify the columns you need in your `SELECT` statement rather than using `SELECT *`. This makes your code more readable and ensures you only retrieve the necessary data.

2. Use table aliases in your SQL queries to make them more concise. For example, instead of writing `SELECT * FROM employees`, you can write `SELECT e.* FROM employees e`.

3. Regularly review and update your queries when the table structure changes to maintain compatibility.

In summary, while `SELECT *` can be safe in some situations, it's generally best practice to explicitly list the columns you need in your SQL queries to improve performance, maintainability, and security.

Reply:
I use `SELECT * FROM {} WHERE` bcz in current table structure we need to get all columns from table to build `Nft` and `NftTransferHistory` structures using `nft_from_row` and  `ransfer_history_from_row`. We had long discussion with @shamardy and decided that such tagle structure will be more convenient for `update some specific field/fields` methods.

So this is our case actually
> Pros of using SELECT *:
Simplicity: It's a convenient way to retrieve all columns from a table without specifying each column explicitly.
Future-proofing: If the table structure changes (e.g., columns are added or removed), SELECT * will automatically adapt to the changes, so your queries won't break.

Reply:
Ok, thanks for clarification 🙂